schema/defs-image.json

```diff
@@ -78,6 +77,17 @@
               }
             }
           }
+        },
+        "annotations": {
+          "id": "https://opencontainers.org/schema/image/annotations",
```

Comment:
This has the same `id` as the defintion below ([here][1] in master).  It would be more DRY to reference that here:

    "annotations": {
      "$ref": "#/definitions/annotations"
    }

[1]: https://github.com/opencontainers/image-spec/blob/00850eca2ab993e282a4921f8b7000b2fcbd26fa/schema/defs-image.json#L84